src/plugin.js

```diff
@@ -107,6 +107,7 @@ class PlaylistMenuItem extends Component {
       throw new Error('Cannot construct a PlaylistMenuItem without an item option');
     }
 
+    playlistItem.showDescription = settings.showDescription;
```

Comment:
What do you think of adding the property on `this` rather than `playlistItem`. 
Reasons:
1. It would seem to make more sense as `this.options_` is directly related to `playlistitem` and not really `options` at all. 
2. `playOnSelect` already does this.

Reply:
Yes I agree with you, I did try that approach but when `createEl()` function was called, I got `this.showDescription = undefined`, I guess in that specific method the scope for `this` was the actual DOM component. The way I managed to pass that value was by sending it as part of Component constructor. Please let me know your thoughts.

Reply:
I was going to suggest an alternative, but looking at where the `PlaylistMenuItem` is used, `playlistItem` is an object with an `item` property. Copying the `showDescription` prop to that object doesn't seem too bad since we're not modifying the item itself which would be `playlistItem.item`.